build failed: fatal error: wasm.h : No such file or directory

## OS Info
```bash
➜  wabt git:(master) cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

➜  wabt git:(master) gcc --version
gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

➜  wabt git:(master) clang --version
clang version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

## build steps

follow the build steps in the [Building using the top-level Makefile (Linux and macOS)](https://github.com/webassembly/wabt#building-using-the-top-level-makefile-linux-and-macos)

make command output:

```bash
➜  wabt git:(master) make gcc-release-no-tests
mkdir -p out/gcc/Release/no-tests/
cd out/gcc/Release/no-tests/ && cmake -G "Unix Makefiles" /home/flyflypeng/Code/Github/wabt/ -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/gcc
-- Check for working C compiler: /usr/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/g++
-- Check for working CXX compiler: /usr/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for alloca.h
-- Looking for alloca.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for snprintf
-- Looking for snprintf - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Check size of size_t
-- Check size of size_t - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Configuring done
-- Generating done
-- Build files have been written to: /home/flyflypeng/Code/Github/wabt/out/gcc/Release/no-tests
make --no-print-directory -C out/gcc/Release/no-tests/ all
...... // above everything is ok
[ 84%] Building CXX object CMakeFiles/wasm.dir/src/wast-parser.cc.o
/home/flyflypeng/Code/Github/wabt/src/wast-parser.cc: In member function ‘wabt::Result wabt::WastParser::ParseConst(wabt::Const*, wabt::WastParser::ConstType)’:
/home/flyflypeng/Code/Github/wabt/src/wast-parser.cc:2262:14: warning: ‘literal’ may be used uninitialized in this function [-Wmaybe-uninitialized]
       result = ParseDouble(literal.type, s, end, &f64_bits);
       ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/flyflypeng/Code/Github/wabt/src/wast-parser.cc:2262:14: warning: ‘end’ may be used uninitialized in this function [-Wmaybe-uninitialized]
/home/flyflypeng/Code/Github/wabt/src/wast-parser.cc:2262:14: warning: ‘s’ may be used uninitialized in this function [-Wmaybe-uninitialized]
[ 85%] Building CXX object CMakeFiles/wasm.dir/src/wat-writer.cc.o
[ 86%] Building CXX object CMakeFiles/wasm.dir/src/interp/binary-reader-interp.cc.o
[ 87%] Building CXX object CMakeFiles/wasm.dir/src/interp/interp.cc.o
[ 87%] Building CXX object CMakeFiles/wasm.dir/src/interp/interp-util.cc.o
[ 88%] Building CXX object CMakeFiles/wasm.dir/src/interp/istream.cc.o
[ 89%] Building CXX object CMakeFiles/wasm.dir/src/interp/interp-wasm-c-api.cc.o
/home/flyflypeng/Code/Github/wabt/src/interp/interp-wasm-c-api.cc:17:10: fatal error: wasm.h: No such file or directory
 #include <wasm.h>
          ^~~~~~~~
compilation terminated.
CMakeFiles/wasm.dir/build.make:1094: recipe for target 'CMakeFiles/wasm.dir/src/interp/interp-wasm-c-api.cc.o' failed
make[3]: *** [CMakeFiles/wasm.dir/src/interp/interp-wasm-c-api.cc.o] Error 1
CMakeFiles/Makefile2:591: recipe for target 'CMakeFiles/wasm.dir/all' failed
make[2]: *** [CMakeFiles/wasm.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make[1]: *** [all] Error 2
Makefile:170: recipe for target 'gcc-release-no-tests' failed
make: *** [gcc-release-no-tests] Error 2
```

So I think should add wasm.h header file in the CMakeList.txt file.

--- a/.github/workflows/build_source_release.yml
+++ b/.github/workflows/build_source_release.yml
@@ -1,0 +1,61 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: archive
+      id: archive
+      run: |
+        VERSION=${{ github.event.release.tag_name }}
+        PKGNAME="wabt-$VERSION"
+        mkdir -p /tmp/$PKGNAME
+        mv * /tmp/$PKGNAME
+        mv /tmp/$PKGNAME .
+        TARBALL=$PKGNAME.tar.xz
+        SHASUM=$PKGNAME.tar.xz.sha256
+        tar cJf $TARBALL $PKGNAME
+        $PKGNAME/scripts/sha256sum.py $TARBALL > $SHASUM
+        echo "::set-output name=tarball::$TARBALL"
+        echo "::set-output name=shasum::$SHASUM"
+
+    - name: upload tarball
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_path: ./${{ steps.archive.outputs.tarball }}
+        asset_name: ${{ steps.archive.outputs.tarball }}
+        asset_content_type: application/gzip
+
+    - name: upload shasum
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ github.event.release.upload_url }}
+        asset_path: ./${{ steps.archive.outputs.shasum }}
+        asset_name: ${{ steps.archive.outputs.shasum }}
+        asset_content_type: text/plain

--- a/.github/workflows/build_source_release.yml
+++ b/.github/workflows/build_source_release.yml
@@ -9,13 +9,7 @@ on:
 jobs:
   build:
     name: build
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    defaults:
-      run:
-        shell: bash
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/setup-python@v1
       with:

